### ISSUE TITLE
[BUG] Codegen 1.27 creates NUnit code that does not compile

### ISSUE THREAD

**Issue by jgraber** [microsoft/playwright#17960]
**Context:**
- Playwright Version: 1.27
- Operating System: Windows
- .NET version: .NET 6
- Browser: Chromium & Firefox
- Extra: -

**Code Snippet**

The new selectors in version 1.27.0 of codegen creates NUnit code in C# that does not compile.

The generated code to click a link looks like this:  

```cs
await Page.GetByRole("link", new() { Name = "https://www.insel.ch" }).ClickAsync();
```

To compile the code, it should look like this:

```cs
await Page.GetByRole(AriaRole.Link, new() { NameString = "https://www.insel.ch" }).ClickAsync();
```

There is “link” instead of AriaRole.Link and “Name” for the property NameString of the PageGetByRoleOptions object.

In codegen version 1.26.0 it created this code that compiled:
```cs 
await Page.Locator("text=https://www.insel.ch").ClickAsync();
```

**Describe the bug**

It looks for me as if the change for the selectors is only partially done in V1.27 and a few cases are missing. You can find the whole generated code in the attached files. (OldVersion.txt is generated with version 1.26 of codegen, CurrentVersion.txt is the output of 1.27 and the manual fixes to make the code compile.)

[OldVersion.txt](https://github.com/microsoft/playwright-dotnet/files/9747537/OldVersion.txt)
[CurrentVersion.txt](https://github.com/microsoft/playwright-dotnet/files/9747538/CurrentVersion.txt)

**Comment by mxschmitt** [comment]
Thats fixed in 1.27.1

**Comment by jgraber** [comment]
Thanks for fixing the bug with the ARIA constants. 

Unfortunately, 1.27.1 still creates a “Name” property for the PageGetByRoleOptions object instead of "NameString". This is much simpler to fix manually, but it would be great if this compile error could also be addressed. 

Is: 
`await Page.GetByRole(AriaRole.Link, new() { Name = "https://www.insel.ch" }).ClickAsync();`
Should:
`await Page.GetByRole(AriaRole.Link, new() { NameString = "https://www.insel.ch" }).ClickAsync();`
